src/components/Weather/WeatherTile/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react'
+import './styles.scss'
+
+import { ThermometerIcon, UmbrellaIcon, WindIcon } from '@entur/icons'
+
+import { useWeather } from '../../../logic'
+import { WeatherIconApi } from '..'
+
+const BREAKPOINTS = {
+    fourItems: 570,
+    threeItems: 380,
+    twoItems: 290,
+}
+
+const BREAKPOINTS_COMPACT = {
+    fourItemsDesktop: 1600,
+    ThreeItemsDesktop: 1246,
+}
+
+interface Props {
+    compact?: boolean
+    className?: string
+}
+
+function WeatherTile(props: Props): JSX.Element {
+    const weather = useWeather()
+
+    const DISPLAY_FOUR_ITEMS_CHRONO = window.innerWidth > BREAKPOINTS.fourItems
```

Comment:
SCREAMING_SNAKE_CASE-formatet brukar me helst berre for konstantar utanfor komponenten. Her kan du godt bruke camelCase, og på alle under.